source/index.html.md

```diff
@@ -98,11 +99,32 @@ The endpoint accepts one parameter and a [Basic authentication](https://en.wikip
 
 `POST https://app.sixpark.com.au/oauth/token`
 
+### Scopes
+
+Name | Description
+--------- | -----------
+questionnaire.questions.read | Read questionnaire question data
+questionnaire.results.read | Read questionnaire result data
+users.read | Read user data
+users.write | Write user data
+users.accounts.read | Read user accounts data
+users.accounts.write | Write user accounts data
+users.accounts.results.read | Read user accounts result data
+users.accounts.portfolio.read | Read user accounts portfolio data
+
 ### FORM Parameters
 
 Parameter | Required | Type | Default | Description
 --------- | ----------- | ----------- | ----------- | -----------
 grant_type | yes | string | `no default` | Must be `client_credentials`
+scope | no | string | `no default` | Must be one or many scopes separated by a ' '
+
+### Query Parameters
```

Comment:
I don't believe this is required as you can use either a `GET` or `POST` with form parameters.

Reply:
yeah just thought I would supply as an option both work but happy to just force one way

Reply:
maybe make it obvious that it's either form or query?